charts/logzio-telemetry/Chart.yaml

```diff
@@ -30,7 +30,7 @@ dependencies:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
+version: 2.2.0
```

Comment:
I think we should bump to 3.x, since we're switching from the processor to the connector, which changes the metrics name if i'm not mistaken.
We also need to add a message about this - that when upgrading it will cause metrics names changes.